### ISSUE TITLE
Plugin doesn't work in Cordova 7.0.1

### ISSUE THREAD

**Issue by dhrytsenko** [dukhanov/cordova-background-timer#3]
Hello! 

I need this plugin, but it does not work for me. I did not receive any event when I tried to use it in my application with Cordoba. It just started with the "OK" response code and that's all. I did not receive any error messages in the error callback. Do you have any plans to work on this or is this plug-in dead?

**Comment by dukhanov** [comment]
Hello,

Could you please send me the code how do you use it?

**Comment by dhrytsenko** [comment]
l try to use your plugin in simple app on Linux machine. This is my environment:

* Cordova version: 7.0.1
* OS: Linux Debian
* Mobile phone: OnePlus 5
* Cordova platform: android 6.2.3
* Installed plugins: 

```
com.skycom.cordova.background-timer 0.0.3 "BackgroundTimer"
cordova-background-geolocation 2.8.5 "BackgroundGeolocation"
cordova-plugin-background-fetch 5.1.1 "CDVBackgroundFetch"
cordova-plugin-dialogs 1.3.3 "Notification"
cordova-plugin-network-information 1.3.3 "Network Information"
cordova-plugin-request-location-accuracy 2.2.1 "Request Location Accuracy"
cordova-plugin-splashscreen 4.0.4-dev "Splashscreen"
cordova-plugin-whitelist 1.3.2 "Whitelist"
cordova.plugins.diagnostic 3.6.5 "Diagnostic"

```

l used your example of config

``` 
var successCallback = function(result) {
   console.log('timer start', result);
}

var errorCallback = function(e) {
   console.log('timer start with error', e);
}

var eventCallback = function(timerEvent) {
    console.log('timer ', timerEvent, ' at time ', new Date());
}

var settings = {
	timerInterval: 60000,
	stopOnTerminate: true 
}

BackgroundTimer.onTimerEvent(eventCallback); // subscribe on timer event
BackgroundTimer.start(successCallback, errorCallback, settings);
````
I didn't use any `hours` and `minutes` cause if l understood correct it should start just from start and fire event every minute. If l did something wrong please clarify it for me

**Comment by dukhanov** [comment]
It looks like it is an issue. 
I will fix it asap.

You can check another timer plugin https://github.com/dukhanov/cordova-simple-timer

**Comment by dukhanov** [comment]
That settings works for me:
```
BackgroundTimer.start(successCallback, errorCallback, {
	timerInterval: 60000,
	startOnBoot: true,
	stopOnTerminate: false,
	hours: 22, // use current time + 1 minute, it looks like a hack, but works with the current version
	minutes: 6,
});
```

**Comment by dhrytsenko** [comment]
Thanks! I will check it tomorrow

**Comment by dhrytsenko** [comment]
Yes, plugin works when i set time in future.

**Comment by dukhanov** [comment]
Got it,
Anyway I will fix it.

**Comment by dukhanov** [comment]
@dhrytsenko I've fixed the issue, check version [0.0.4](https://github.com/dukhanov/cordova-background-timer/releases/tag/0.0.4)

**Comment by dhrytsenko** [comment]
Thank you! It works now!
